src/Angor/Client/Pages/Wallet.razor

```diff
@@ -596,8 +607,20 @@
             sendConfirmModal = false;
 
             var wallet = _walletStorage.GetWallet();
+            var accountInfo = storage.GetAccountInfo(network.Name);
+            var unconfirmedInfo = _cacheStorage.GetUnconfirmedInfo();
+
+            var res = await _walletOperations.SendAmountToAddress(wallet, _sendInfo);
 
-            return await _walletOperations.SendAmountToAddress(wallet, _sendInfo);
+            if (res.Success)
+            {
+                _walletOperations.UpdateAccountUnconfirmedInfoWithSpentTransaction(accountInfo, unconfirmedInfo, res.Data);
+                accountBalanceInfo = AccountBalanceInfo.GetBalance(accountInfo, unconfirmedInfo);
```

Comment:
The name suggests that the account balance info holds the state end returns the balance not that it is a static method